doc/devel/documenting_mpl.rst

```diff
@@ -489,7 +499,7 @@ result in a lot of whitespace within the line):
       Parameters
       ----------
       projection : {'aitoff', 'hammer', 'lambert', 'mollweide', 'polar', \
-  'rectilinear'}, optional
+  'rectilinear'}
```

Comment:
Did I misunderstand - shouldn't this have a default value or `, optional` at the end, now?

Reply:
Thanks, fixed.